### PR TITLE
Add chrome stable to dockerfile and web shard

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Adds a download of chrome stable to the linux dockerfile and adds a new linux shard to run the passing subset of web tests

### CODE REVIEW

**Comment by jonahwilliams** [comment]
This will have to wait until the web tests can be relanded

File: .cirrus.yml [hunk 1]
```diff
@@ -76,6 +76,9 @@ task:
       container:
         cpu: 4
         memory: 12G
+    - name: web_tests-linux
```
Comment (jonahwilliams) [inline .cirrus.yml]: This doesn't seem to be quite enough...

Comment (gspencergoog) [reply .cirrus.yml]: What happens/doesn't happen?

**Comment by gspencergoog** [comment]
Did you rebuild and update the Dockerfile?

**Comment by jonahwilliams** [comment]
I've rebuilt the image but I do not have permission to upload. Which buckets/permissions do I need?

**Comment by jonahwilliams** [comment]
Resolved: problem was in chair

**Review by dnfield — APPROVED** [review]
LGTM

**Pull request merged**